src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -2,6 +2,69 @@
 
 public class SalaryInfo {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line at the start of the class definition.